.github/workflows/onPushToMain.yml

```diff
@@ -0,0 +1,55 @@
+name: version, tag and github release
```

Comment:
@wileyj the `oclif generate ..` command used to update the CLI libraries added these release automation workflows. The code looks fine to me, but some repo secrets need to be added for it to work. Thoughts?